lib/resty/etcd/v3.lua

```diff
@@ -30,7 +32,53 @@ local mt = { __index = _M }
 -- define local refresh function variable
 local refresh_jwt_token
 
-local function _request_uri(self, method, uri, opts, timeout, ignore_auth)
+local function incr(key, shm_name, failure_window)
```

Comment:
for this function, we need a better name

Reply:
update, pls review it